Answers.md

```diff
@@ -1,11 +1,18 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+		is a JS library for building UI it's great for single page websites the use of DOM is expensive in terms of performance and too much of it makes the app slow React changes on the fly and allows for smoother and fast app useage
 
 1.  What does it mean to _think_ in react?
+		break up the UI into different pieces, make a static version, see where the "state" goes, and then add data flow

```

Comment:
You don't necessarily need to make it a static version, but you do want to break it down into component pieces. So you want a component layout. Everything else is correct